main.js

```diff
@@ -184,5 +184,7 @@ $(function() {
       return false;
     }
     hasLoaded = true;
-  })();
+  }
+
+  $(document).ready(collapseRanges);
```

Comment:
I doubt this is really necessary because the code is already wrapped in a jquery "ready" handler.